mantidproject/mantid
# Update FDA plotting and fitting

**Description of work.**

The frequency domain analysis GUI has numerous usability problems. This PR fixes enough of the issues that it is possible to perform transforms on time domain data and then perform fits on the transformed values but there is more work still to be done.

**To test:**
 * Test that you can perform frequency domain transforms and perform fits on single data workspaces. Check that the plots update in the expected manner when you do so. Get in touch with me to explain the workflows in this GUI is you are not already familiar with it.



Fixes 28742 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
